tests/41end-to-end-keys/07-backup.pl

```diff
@@ -0,0 +1,271 @@
+my $fixture = local_user_fixture();
+
+my $current_version; # FIXME: is there a better way of passing the backup version between tests?
+
+test "Can create backup version",
+   requires => [ $fixture ],
+
+   proves => [qw( can_create_backup_version )],
+
+   do => sub {
+      my ( $user ) = @_;
+
+      do_request_json_for( $user,
+         method  => "POST",
+         uri     => "/unstable/room_keys/version",
+         content => {
+            algorithm => "m.megolm_backup.v1",
```

Comment:
hm, shouldn't we use a real algorithm for this? (or is this a real one? it surprises me it doesn't include info about the ciphers being used, like the m.room.encryption algorithm does)

Reply:
That's the algorithm given in the proposal, but I agree, it should probably include some algorithm info